lib/config/migration.spec.ts

```diff
@@ -477,88 +477,6 @@ describe('config/migration', () => {
       });
     });
 
-    it('it migrates unpublishSafe', () => {
```

Comment:
Is it necessary to remove these tests? Doesn't keeping them allow us to verify backwards compatibility of this refactor?

Reply:
It is not necessary, but in this case unit tests will be duplicated

Reply:
It's good to keep these types of tests during refactoring so that we know that they still pass the same. Then once refactoring is done you can probably remove nearly all of them in one go. For now, please restore them so that this PR is easier to review/approve

Reply:
Make sense, unit tests restored